### PR TITLE
ComicExtra Extension Source

### PR DESCRIPTION
**Pull request by Yong-L**
Addresses #390

### CODE REVIEW

File: src/en/comicextra/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+ext {
+    appName = 'Tachiyomi: ComicExtra'
+    pkgNameSuffix = 'en.comicextra'
+    extClass = '.ComicExtra'
+    extVersionCode = 2
+    libVersion = '1.2'
+}
+
+dependencies {
```
Comment (arkon) [inline src/en/comicextra/build.gradle]: You can remove these if you're not using them.

File: src/en/comicextra/build.gradle [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    appName = 'Tachiyomi: ComicExtra'
+    pkgNameSuffix = 'en.comicextra'
+    extClass = '.ComicExtra'
+    extVersionCode = 2
```
Comment (arkon) [inline src/en/comicextra/build.gradle]: Start at 1.

Comment (Yong-L) [reply src/en/comicextra/build.gradle]: Done. Should I also change the libversion to 1.0 as well?

Comment (arkon) [reply src/en/comicextra/build.gradle]: Nope, using the latest version of the library is preferred.

**Pull request merged**